source/rest/manager/src/Api/ConfigApi.php

```diff
@@ -32,39 +32,12 @@ protected function load()
   }
 
   protected function dump() {
-    $out =  <<<EOD
+    $out =  '
 <?php
-// File for configurations that shouldn't be shared with the user
-\$hidden = array(
-
-EOD;
-    $first = true;
-    foreach( $this->hidden as $section => $options )
-    {
-      if (!$first) {
-        $out .= ",\n";
-      } else {
-        $first = false;
-      }
-      $out .= "  '$section' => array(";
-
-      $firstKey = true;
-        foreach( $options as $key => $value )
-        {
-          if( !$firstKey ) {
-            $out .= ', ';
-          } else {
-            $firstKey = false;
-          }
-          $out .= "'" . $key . "' => '" . $value . "'";
-        }
-      $out .= ')';
-    }
-    $out .= "\n);\n?>";
-
-    print_r($this->hidden);
-
-
+// File for configurations that shouldn\'t be shared with the user
+$data = \'' . json_encode($this->hidden, JSON_PRETTY_PRINT) . '\';
```

Comment:
This should also have the option `JSON_UNESCAPED_SLASHES` to keep configured URLs readable (and probably also `JSON_PRESERVE_ZERO_FRACTION` to prevent modification of data that is not intended by the user, just in case)